pyupgrade.py

```diff
@@ -1593,13 +1680,13 @@ def _replace(i, mapping, node):
             j = _find_open_paren(tokens, i)
             func_args, end = _parse_call_args(tokens, j)
             node = visitor.six_calls[token.offset]
-            template = _get_tmpl(SIX_CALLS, node.func)
+            template = _get_tmpl(SIX_CALLS, cast(ast.Attribute, node.func))
```

Comment:
this isn't always an `Attribute` right? I thought it was `Union[Name, Attribute]`

Reply:
urgh let me think on this..

Reply:
yea